seaborn_objects_recipes/recipes/plotting.py

```diff
@@ -61,4 +63,62 @@ def plot(self, data, xvar, yvar):
         
         
         return plot
-    
\ No newline at end of file
+
+
+
+@dataclass
+class PolyFit(Stat):
+    """
+    Fit a polynomial of the given order and resample data onto predicted curve
+    including confidence intervals.
+    """
+    alpha: float = 0.05
+    order: int = 2
+    gridsize: int = 100
+    num_bootstrap: Optional[int] = None
```

Comment:
For a regular regression, probably better to get CIs directly from the analytic result (rather than bootstrap), but that's a side issue. 

Looking at this, looks like you just updated `PolyFit` to do standard errors?

Reply:
I tried that option and was getting CIs close to the actual values so it wasn't showing properly in the plot.

![IMG_0267](https://github.com/Ofosu-Osei/seaborn_objects_recipes/assets/6976037/9ee1b999-26e9-428f-bd64-76eaa99567ae)

Reply:
You should be getting roughly the same SEs with either strategy, so if they seem really different something's probably off in one implementation or the other. They may vary a little, but not by much (and you could always just change your `\alpha` threshold to get them wider for plotting for an example image)

Reply:
Alright then, I'll make the changes accordingly.